target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -146,6 +146,10 @@ glinet,gl-ar300m-lite)
 glinet,gl-ar300m16)
 	ucidef_set_led_netdev "lan" "LAN" "gl-ar300m:green:lan" "eth0"
 	;;
+glinet,gl-mifi)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:green:lan" "eth0"
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
```

Comment:
For correct link detection, typically one of the interfaces needs to be set up with ucidef_set_led_switch. I'm never sure which, though. (Take this as optional improvement.)

Reply:
After some fiddling I found out the switch offset for the external lan port and changed led to use that in 21e0a0d.